### PR TITLE
Possible new organization for Docs

### PR DESCRIPTION
**Pull request by BabylonJSGuide**
Only @PirateJC and @RaananW should merge.

May contain errors in links and redirects.

Workflow section folders and files for organization possibilities only, content for this area is not my strong point.

### CODE REVIEW

**Comment by BabylonJSGuide** [comment]
@RaananW 

Ref #560 
In this draft I have already pulled Contibuting to the top level. 

If it is possible, what do you think of deploying this draft to give you and Jason and others who have suggestions a better idea of what the draft looks like?

File: lib/buildUtils/typedoc.utils.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ export const generateTypeDoc = async () => {
     if (!existsSync(basePathResolved + sep + "files" + sep + "index.html")) {
         console.log("generating API docs, patience is required");
         // download the latest .d.ts
-        const response = await fetch("https://preview.babylonjs.com/documentation.d.ts");
+        const response = await fetch("https://raw.githubusercontent.com/BabylonJS/Babylon.js/master/legacy/dist/preview%20release/documentation.d.ts");
```
Comment (RaananW) [inline lib/buildUtils/typedoc.utils.ts]:
```suggestion
        const response = await fetch("https://preview.babylonjs.com/documentation.d.ts");
```

File: vercel.json [hunk 2]
```diff
@@ -11,235 +11,550 @@
...
+        { "source": "/advanced_topics/mathShadows", "destination": "/features/divingDeeper/lights/mathShadows" },
+        { "source": "/advanced_topics/minifiedVer", "destination": "/workflow/wfDeeper/minifiedVer" },
+        { "source": "/advanced_topics/mutliPlatTextures", "destination": "/features/divingDeeper/materials/advanced/" },
+        { "source": "/advanced_topics/renderPasses", "destination": "/features/divingDeeper/materials/advanced/renderPasses" },
+        { "source": "/advanced_topics/serverSide", "destination": "/features/advanced_topics/serverSide" },
+        { "source": "/advanced_topics/Shader_Material", "destination": "/features/divingDeeper/materials/shaders/shaderMaterial" },
+        { "source": "/advanced_topics/shaders", "destination": "/features/divingDeeper/materials/shaders" },
+        { "source": "/advanced_topics/shaders/computeShader", "destination": "/features/divingDeeper/materials/shaders/computeShader" },
+        { "source": "/advanced_topics/shaders/Fog+ShaderMat", "destination": "/features/divingDeeper/materials/shaders/Fog+ShaderMat" },
```
Comment (RaananW) [inline vercel.json]:
```suggestion
        { "source": "/advanced_topics/shaders/Fog\+ShaderMat", "destination": "/features/divingDeeper/materials/shaders/Fog\+ShaderMat" },
```

**Review by RaananW — COMMENTED** [review]
trying to fix the build issue to have vercel deploy the branch

File: vercel.json [hunk 3]
```diff
@@ -11,235 +11,550 @@
...
+        { "source": "/advanced_topics/mathShadows", "destination": "/features/divingDeeper/lights/mathShadows" },
+        { "source": "/advanced_topics/minifiedVer", "destination": "/workflow/wfDeeper/minifiedVer" },
+        { "source": "/advanced_topics/mutliPlatTextures", "destination": "/features/divingDeeper/materials/advanced/" },
+        { "source": "/advanced_topics/renderPasses", "destination": "/features/divingDeeper/materials/advanced/renderPasses" },
+        { "source": "/advanced_topics/serverSide", "destination": "/features/advanced_topics/serverSide" },
+        { "source": "/advanced_topics/Shader_Material", "destination": "/features/divingDeeper/materials/shaders/shaderMaterial" },
+        { "source": "/advanced_topics/shaders", "destination": "/features/divingDeeper/materials/shaders" },
+        { "source": "/advanced_topics/shaders/computeShader", "destination": "/features/divingDeeper/materials/shaders/computeShader" },
+        { "source": "/advanced_topics/shaders/Fog\+ShaderMat", "destination": "/features/divingDeeper/materials/shaders/Fog\+ShaderMat" },
```
Comment (RaananW) [inline vercel.json]:
```suggestion
        { "source": "/advanced_topics/shaders/FogShaderMat", "destination": "/features/divingDeeper/materials/shaders/FogShaderMat" },
```

File: vercel.json [hunk 4]
```diff
@@ -278,453 +593,306 @@
...
+        { "source": "/resources/BabylonJS_and_PixiJS", "destination": "/workflow/Babylon.js+ExternalLibraries/BabylonJS_and_PixiJS" },
+        { "source": "/resources/babylonjs_and_pixijs", "destination": "/workflow/Babylon.js+ExternalLibraries/BabylonJS_and_PixiJS" },
+        { "source": "/resources/BabylonJS_and_React_Native", "destination": "/workflow/Babylon.js+ExternalLibraries/BabylonJS_and_React_Native" },
+        { "source": "/resources/babylonjs_and_react_native", "destination": "/workflow/Babylon.js+ExternalLibraries/BabylonJS_and_React_Native" },
+        { "source": "/resources/BabylonJS_and_ReactJS", "destination": "/workflow/Babylon.js+ExternalLibraries/BabylonJS_and_ReactJS" },
+        { "source": "/resources/babylonjs_and_reactjs", "destination": "/workflow/Babylon.js+ExternalLibraries/BabylonJS_and_ReactJS" },
+        { "source": "/resources/BabylonJS_and_VRM", "destination": "/workflow/Babylon.js+ExternalLibraries/BabylonJS_and_VRM" },
+        { "source": "/resources/babylonjs_and_vrm", "destination": "/workflow/Babylon.js+ExternalLibraries/BabylonJS_and_VRM" },
```
Comment (RaananW) [inline vercel.json]:
```suggestion
```

File: vercel.json [hunk 5]
```diff
@@ -278,453 +593,306 @@
...
+        { "source": "/resources/projects", "destination": "/toolsAndResources/Unity/Projects" },
+        { "source": "/resources/Renaming_Project", "destination": "/toolsAndResources/Editor/Workspace/Renaming_Project" },
+        { "source": "/resources/renaming_project", "destination": "/toolsAndResources/Editor/Workspace/Renaming_Project" },
+        { "source": "/resources/Rotation_Conventions", "destination": "/features/divingDeeper/mesh/transforms/center_origin/rotation_conventions" },
+        { "source": "/resources/rotation_conventions", "destination": "/features/divingDeeper/mesh/transforms/center_origin/rotation_conventions" },
+        { "source": "/resources/rotation_conventions", "destination": "/features/divingDeeper/mesh/transforms/center_origin/rotation_conventions" },
+        { "source": "/resources/Running_a_local_webserver_for_BabylonJs", "destination": "/workflow/Babylon.js+ExternalLibraries/Running_a_local_webserver_for_BabylonJs" },
+        { "source": "/resources/running_a_local_webserver_for_babylonjs", "destination": "/workflow/Babylon.js+ExternalLibraries/Running_a_local_webserver_for_BabylonJs" },
```
Comment (RaananW) [inline vercel.json]:
```suggestion
```

File: vercel.json [hunk 6]
```diff
@@ -11,235 +11,550 @@
...
+        { "source": "/extensions/Babylon.js+ExternalLibraries/BabylonJS_and_Ionic_Angular", "destination": "/workflow/Babylon.js+ExternalLibraries/BabylonJS_and_Ionic_Angular" },
+        { "source": "/extensions/Babylon.js+ExternalLibraries/BabylonJS_and_PixiJS", "destination": "/workflow/Babylon.js+ExternalLibraries/BabylonJS_and_PixiJS" },
+        { "source": "/extensions/Babylon.js+ExternalLibraries/BabylonJS_and_React_Native", "destination": "/workflow/Babylon.js+ExternalLibraries/BabylonJS_and_React_Native" },
+        { "source": "/extensions/Babylon.js+ExternalLibraries/BabylonJS_and_ReactJS", "destination": "/workflow/Babylon.js+ExternalLibraries/BabylonJS_and_ReactJS" },
+        { "source": "/extensions/Babylon.js+ExternalLibraries/BabylonJS_and_VRM", "destination": "/workflow/Babylon.js+ExternalLibraries/BabylonJS_and_VRM" },
+        { "source": "/extensions/Babylon.js+ExternalLibraries/Running_a_local_webserver_for_BabylonJs", "destination": "/workflow/wfDeeper/Running_a_local_webserver_for_BabylonJs" },
```
Comment (RaananW) [inline vercel.json]:
```suggestion
```

File: configuration/structure.json [hunk 7]
```diff
@@ -1,889 +1,1164 @@
...
+                        "children": {
+                            "intro": {
+                                "friendlyName": "Introduction",
+                                "content": "How_to/GUI/gui_editor_intro",
+                                "children": {}
+                            },
+                            "guiUsage": {
+                                "friendlyName": "Usage",
+                                "children": {},
+                                "content": "How_to/GUI/gui_editor_usage"
```
Comment (RaananW) [inline configuration/structure.json]:
```suggestion
                                "content": "How_To/GUI/gui_editor_intro",
                                "children": {}
                            },
                            "guiUsage": {
                                "friendlyName": "Usage",
                                "children": {},
                                "content": "How_To/GUI/gui_editor_usage"
```

File: configuration/structure.json [hunk 8]
```diff
@@ -1,889 +1,1164 @@
...
-                                "pgTemplates": { "friendlyName": "Playground Templates", "children": {}, "content": "resources/Playground_aspects/playgroundTemplates" },
-                                "yourOwnSnippetServer": {"friendlyName": "Make your own Snippet Server", "children": {}, "content": "resources/Playground_aspects/yourOwnSnippetServer"}
+                            "assets": {
+                                "friendlyName": "Assets",
+                                "children": {
+                                    "addingMeshes": { "friendlyName": "Adding Meshes", "children": {}, "content": "resources/Editor/Assets/Adding_Meshes" },
+                                    "addingMaterials": { "friendlyName": "Adding Materials", "children": {}, "content": "resources/Editor/Assets/Adding_Materials" },
+                                    "addingTextures": { "friendlyName": "Adding Textures", "children": {}, "content": "resources/Editor/Assets/Adding_Textures" },
+                                    "addingSounds": { "friendlyName": "Adding Sounds", "children": {}, "content": "resources/Editor/Assets/Adding_Sounds" }
```
Comment (RaananW) [inline configuration/structure.json]:
```suggestion
                                    "addingSounds": { "friendlyName": "Adding Sounds", "children": {}, "content": "resources/Editor/Assets/Adding_Textures" }
```
The file is missing, temporarily setting to a different file

File: configuration/structure.json [hunk 9]
```diff
@@ -1165,99 +1299,92 @@
...
+                        "friendlyName": "To The Documentation",
+                        "content": "How_To/contribute/contribute_to_documentation", 
+                        "children": {
+                            "easydoc": {
+                                "friendlyName": "Minor Edits",
+                                "content": "How_to/contribute/easydoc"
+                            },
+                            "addToDocs": {
+                                "friendlyName": "Major Additions",
+                                "content": "How_to/contribute/add_to_docs"
```
Comment (RaananW) [inline configuration/structure.json]:
```suggestion
                                "content": "How_To/contribute/easydoc"
                            },
                            "addToDocs": {
                                "friendlyName": "Major Additions",
                                "content": "How_To/contribute/add_to_docs"
```

**Comment by PirateJC** [comment]
Closing this PR as it is being addressed with the following PR:

https://github.com/BabylonJS/Documentation/pull/622

**Pull request closed without merging**